x/epoching/keeper/keeper.go

```diff
@@ -107,7 +107,7 @@ func (k Keeper) GetEpochMsg(ctx sdk.Context, epochNumber int64, actionID uint64)
 // GetEpochActions get all actions
 func (k Keeper) GetEpochActions(ctx sdk.Context) []sdk.Msg {
 	actions := []sdk.Msg{}
-	iterator := k.GetEpochActionsIterator(ctx)
+	iterator := sdk.KVStorePrefixIterator(ctx.KVStore(k.storeKey), EpochActionQueuePrefix)
```

Comment:
Why is this in this PR, nothing here is tendermint related?

Reply:
My suspicion based on some poking at things, is that github's diff is just overwhelmed and that this is just a change to master that leaked into the PR. 

If you do `git diff master` in this branch, this doesn't show up.

Reply:
ah gotcha

Reply:
But I still think we need to revert this in this PR, right? On master, we currently have:

https://github.com/cosmos/cosmos-sdk/blob/18eb33fba006736ed5d29ae3e7a1326575faebad/x/epoching/keeper/keeper.go#L110